html/changelog.html

```diff
@@ -1,60 +1,73 @@
-  <base target="_blank" />
-  <script type='text/javascript'>
-  
-	function changeText(tagID, newText, linkTagID){
-		var tag = document.getElementById(tagID);
-		tag.innerHTML = newText;
-		var linkTag = document.getElementById(linkTagID);
-		linkTag.removeAttribute("href");
-		linkTag.removeAttribute("onclick");
-	}
-	
-  </script>  
-<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-</head>
-
-<body>
-<table align='center' width='650'><tr><td>
-<table align='center' class="top">
-	<tr>
-		<td valign='top'>
-			<div align='center'><font size='3'><b>Space Station 13</b></font></div>
-			
-			<p><div align='center'><font size='3'><a href="https://aurorastation.org/wiki/index.php?title=Main_Page">Wiki</a> | <a href="https://github.com/Aurorastation/Aurora.3">Source code</a></font></div></p>
-            <font size='2'>Code licensed under <a href="http://www.gnu.org/licenses/agpl.html">AGPLv3</a>. Content licensed under <a href="http://creativecommons.org/licenses/by-sa/3.0/">CC BY-SA 3.0</a>.<br><br>
-			</td>
-	</tr>
-</table>
-
-<br><b>Baystation 12 / Aurorastation Credit List</b>
-<table align='center' class="top">
-	<tr>
-		<td valign='top'>
-            <font size='2'><b>Current Project Maintainers:</b> <a href='https://github.com/orgs/Aurorastation/people'>-Click Here-</a><br></font>
-			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/Aurorastation/Aurora.3/graphs/contributors'>-Click Here-</a><br></font>
-			<font size='2'><b>Code:</b> Abi79, Aryn, Cael_Aislinn, Ccomp5950, Chinsky, cib, CompactNinja, DopeGhoti, Erthilo, Hawk_v3, Head, Ispil, JoeyJo0, Lexusjjss, Melonstorm, Miniature, Mloc, NerdyBoy1104, PsiOmegaDelta, SkyMarshal, Snapshot, Spectre, Strumpetplaya, Sunfall, Tastyfish, Uristqwerty<br></font>
-			<font size='2'><b>Sprites:</b> Apple_Master, Arcalane, Chinsky, CompactNinja, Deus Dactyl, Erthilo, Flashkirby, JoeyJo0, Miniature, Searif, Xenone, faux<br></font>
-			<font size='2'><b>Sounds:</b> Aryn<br></font>
-            <font size='2'><b>Main Testers:</b> Anyone who has submitted a bug to the issue tracker<br></font>
-			<font size='2'><b>Thanks to:</b> /tg/ station, /vg/station, GoonStation devs, Baystation 12 devs, the original SpaceStation developers and FFrances for the title image.<br> Also a thanks to anybody who has contributed who is not listed here :( Ask to be added here on irc.</font>
-			<font size='2' color='red'><b><br>Have a bug to report?</b> Visit our <a href="https://github.com/Aurorastation/Aurora.3/issues">Issue Tracker</a>.<br></font>
-		</td>
-	</tr>
-</table>
-
-		<!--
-		
-		TO ADD AN ENTRY, ADD AND MAINTAIN YOUR OWN changelog/USERNAME.yml FILE. 
-		
-		*** DO NOT MODIFY THIS FILE OR YOU WILL CAUSE MERGE CONFLICTS. ***
-		
-		-->
-		<div class="commit sansserif">
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Aurorastation Changelog</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
This line should be before anything. Because it has to go back and reparse what it parsed.